README.rst

```diff
@@ -64,64 +77,69 @@ Then, run pcluster configure:
     subnet-b921nv04
   Master Subnet ID []:
 
-Now you can create your first cluster;
+Now create your first cluster stack:
 
 .. code-block:: sh
 
   $ pcluster create myfirstcluster
 
-
-After the cluster finishes creating, log in:
+After the cluster creation process finishes, login to the head node:
 
 .. code-block:: sh
 
   $ pcluster ssh myfirstcluster
 
-You can view the running compute hosts:
+View the running compute hosts based on the selected scheduler:
 
 .. code-block:: sh
 
-  $ qhost
+  $ qhost                              [ Grid Engine ]
+  $ pbsnodes -a                        [ Torque ]
+  $ sinfo -N                           [ Slurm ]
+  $ awsbhosts -c myfirstcluster -d     [ AWS Batch ]
 
-For more information on any of these steps see the `Getting Started Guide`_.
+For more information on any of these steps, please refer to the `Getting Started Guide`_.
 
 .. _`Getting Started Guide`: https://aws-parallelcluster.readthedocs.io/en/latest/getting_started.html
 
+-------------
 Documentation
 -------------
 
-Documentation is part of the project and is published to -
-https://aws-parallelcluster.readthedocs.io/. Of most interest to new users is
-the Getting Started Guide -
-https://aws-parallelcluster.readthedocs.io/en/latest/getting_started.html.
+Documentation for AWS ParallelCluster can be found by visiting the project page:
+https://aws-parallelcluster.readthedocs.io/
+
+New users are strongly encouraged to review the Getting Started Guide:
+https://aws-parallelcluster.readthedocs.io/en/latest/getting_started.html
 
+------
 Issues
 ------
 
-Please open a GitHub issue for any feedback or issues:
-https://github.com/aws/aws-parallelcluster.  There is also an active AWS
-HPC forum which may be helpful:https://forums.aws.amazon.com/forum.jspa?forumID=192.
+Please visit the AWS ParallelCluster Github project site to provide feedback, request new features, or report bugs:
+https://github.com/aws/aws-parallelcluster.
 
+The AWS HPC Forum is monitored by the ParallelCluster development team and may also be helpful:
+https://forums.aws.amazon.com/forum.jspa?forumID=192.
+
+-------
 Changes
 -------
 
 CfnCluster 1.6 IAM Change
 =========================
-Between CfnCluster 1.5.4 and 1.6.0 we made a change to the CfnClusterInstancePolicy that adds “s3:GetObject” permissions
-on objects in <REGION>-cfncluster bucket, "autoscaling:SetDesiredCapacity", "autoscaling:DescribeTags" permissions and
+Between CfnCluster 1.5.4 and 1.6.0, we made a change to the CfnClusterInstancePolicy that adds “s3:GetObject” permissions
```

Comment:
I know that is not related to your patch, but could you also change the `“` surrounding the s3:GetObject to `"` ?

Reply:
Done.